docs/topics/templates/document-attributes.adoc

```diff
@@ -56,20 +69,19 @@
 // Commenting out as these attributes are not currently used:
 // :ProductRelease: 4
 // :ProductVersion: 4.0
-:ProductDistributionVersion: 5.2.1.Final
+:ProductDistributionVersion: 5.3.0.Final
 :ProductDistribution: mta-cli-{ProductDistributionVersion}
-:MavenProductVersion: 5.2.1.Final
+:MavenProductVersion: 5.3.0.Final
```

Comment:
Same here @PhilipCattanach The Maven product version won't be 5.3 right ?

Reply:
Correct. It should be 1.0.0